dan-web/.env

```diff
@@ -1,2 +1,5 @@
 NODE_PATH=src
 REACT_APP_GITHUB_URL = https://github.com/fiatrete/DAN-Stable-Diffusion-Computing-Network
+REACT_APP_BASE_API_URL = https://api.opendan.ai
+REACT_APP_COOKIE_DOMAIN = opendan.ai
+REACT_APP_GOOGLE_ANALYTICS_ID = ''
```

Comment:
REACT_APP_GOOGLE_ANALYTICS_ID need to be enclosed in single quotes? 
you may change it to 
```
REACT_APP_GOOGLE_ANALYTICS_ID = YOUR_GOOGLE_ANALYTICS_ID
```

Reply:
I have made the change and recommitted.

Reply:
I have made the change and recommitted.